Document how to bump dependencies (#446)
ansible/ansible-documentation: README.md

## Building Ansible community documentation

Follow the documentation to [set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally) and then [build Ansible community documentation locally](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#building-the-documentation-locally)

## Updating the dependencies

To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image:
```bash

> podman run --rm --tty --volume "$(pwd):/mnt" --workdir /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'

Reviewer: ```suggestion
podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt python:3.10 bash -c 'pip install nox ; nox -e pip-compile'
```

## Building Ansible community documentation

Follow the documentation to [set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally) and then [build Ansible community documentation locally](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#building-the-documentation-locally)

## Updating the dependencies

> To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image: `podman run --rm -t -v "$(pwd):/mnt" -w /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'`.

Author: `-t` can also be removed, but then the console output is less fancy (no colors for me, for example).

Reviewer 2: how about changing `-t` to `--tty` so it's more apparent what that option is doing?

Author: Good point, let's write out all the options for clarity:
```suggestion
To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image: `podman run --rm --tty --volume "$(pwd):/mnt" --workdir /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'`.
```

> To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image: `podman run --rm -it -v "$(pwd):/mnt" -w /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'`.

Author: ```suggestion
To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image: `podman run --rm -t -v "$(pwd):/mnt" -w /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'`.
```

> To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image: `podman run --rm --tty --volume "$(pwd):/mnt" --workdir /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'`.

Reviewer: Can you put the command in a markdown code block so it can be copy-pasted more easily?

Author: Sure!
````suggestion
To update dependencies, you can use `nox -e pip-compile`. Since this requires Python 3.10, this might not work in your environment if you do not have Python 3.10 installed. In that case, you can use root-less podman with a Python 3.10 image:
```bash
podman run --rm --tty --volume "$(pwd):/mnt" --workdir /mnt python:3.10 /bin/bash -c 'pip install nox ; nox -e pip-compile'
```
````
